pkgs/tools/misc/empty/fix-cc-parameter-order.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/Makefile b/Makefile
+index a198039..c87b5d5 100644
+--- a/Makefile
++++ b/Makefile
+@@ -18,7 +18,7 @@ BINDIR = $(PREFIX)/bin
+ MANDIR = $(PREFIX)/man/man1
+ 
+ all:
+-	${CC} ${CFLAGS} -Wall ${LIBS} -o empty empty.c
++	${CC} ${CFLAGS} -Wall -o empty empty.c ${LIBS}
```

Comment:
This patch can also be easily replaced by substituteInPlace.

Reply:
Done.